app/open_ai_client.rb

```diff
@@ -1,11 +1,18 @@
 require 'openai'
+require 'json'
 
 class OpenAIClient
   def initialize
     @client = OpenAI::Client.new
     @model = ENV.fetch('OPEN_AI_MODEL', "gpt-3.5-turbo")
     @temperature = ENV.fetch('OPEN_AI_TEMPERATURE', 0.1)
-    @system_prompt = ENV.fetch('SYSTEM_PROMPT', system_prompt_default)
+
+    prompts = JSON.parse(File.read("#{ENV['GITHUB_WORKSPACE']}/prompts.json"))
```

Comment:
Not sure if `@prompts` is the right variable name as the file will also have to deal with other variables